### ISSUE TITLE
log output format

### ISSUE THREAD

**Issue by rongrongxiangxin** [dmlc/mxnet#2254]
Some examples' output like this:
INFO:root:Start training with [gpu(0)]
INFO:root:Epoch[0] Batch [200]  Speed: 70941.64 samples/sec Train-accuracy=0.389050
INFO:root:Epoch[0] Batch [400]  Speed: 97857.94 samples/sec Train-accuracy=0.646450
INFO:root:Epoch[0] Batch [600]  Speed: 70507.97 samples/sec Train-accuracy=0.743333
INFO:root:Epoch[0] Resetting Data Iterator
INFO:root:Epoch[0] Train-accuracy=0.743333
INFO:root:Epoch[0] Time cost=1.069
INFO:root:Epoch[0] Validation-accuracy=0.950800

However, I got the output like this:
INFO:root:Start training with [gpu(0)]
INFO:root:Epoch[0] Resetting Data Iterator
INFO:root:Epoch[0] Time cost=1.069
INFO:root:Epoch[0] Validation-accuracy=0.950800

There were only Validation-accuracy.  I did not find where to set the output format.

**Comment by piiswrong** [comment]
use batch_end_callback for the first kind of output

**Comment by rongrongxiangxin** [comment]
@piiswrong thanks for your reply.  I have used 'batch_end_callback '.
My codes are as follows:
    model = mx.model.FeedForward(
        ctx                = devs,
        symbol             = network,
        num_epoch          = args.num_epochs,
        initializer        = mx.init.Load(params,default_init=mx.init.Xavier(factor_type="in", magnitude=2.34)),
        #initializer        = mx.init.Xavier(factor_type="in", magnitude=2.34),
        optimizer          = optimizer,
        **model_args)

    import logging
    head = '%(asctime)-15s %(message)s'
    logging.basicConfig(level=logging.DEBUG,format=head)

    model.fit(
        X                  = train,
        eval_data          = val,
        kvstore            = kv,
        eval_metric        = ['accuracy'],
        batch_end_callback = mx.callback.Speedometer(args.batch_size, 50),
        epoch_end_callback = checkpoint)

The code is about finetune based on the pretraind net. I am very confused about the output.
